test/e2e/scheduling/priorities.go

```diff
@@ -210,7 +210,12 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		}
 	})
 
-	ginkgo.It("Pod should be preferably scheduled to nodes pod can tolerate", func() {
+	/*
+		Release : v1.16
+		Testname: Scheduler, Pod Toleration
+		Description: Pod MUST be preferably scheduled onto the Node whose taint can be tolerated it.
```

Comment:
s/tolerated it/tolerated./

Reply:
Hope I have resolved the concern as per expectation. Please do let me know if otherwise